examples.md

```diff
@@ -353,6 +354,23 @@ The yarn cache directory will depend on your operating system and version of `ya
       ${{ runner.os }}-yarn-
 ```
 
+
+## Node - Pnpm
+
+```yaml
+  - name: Cache .pnpm-store
+    uses: actions/cache@v2
+    with:
+      path: |
+      	 ~/.pnpm-store
+         D:\.pnpm-store
```

Comment:
these pnpm store paths could be resolved from running `pnpm store path --silent`, as is done in `actions/setup-node`:

- https://github.com/actions/setup-node/blob/26961cf329f22f6837d5f54c3efd76b480300ace/src/cache-utils.ts#L35